contrib/podmanremoteimage/README.md

```diff
@@ -0,0 +1,26 @@
+podman-remote-images
+====================
+
+Overview
+--------
+
+This directory contains the containerfile for creating a container image which consist podman-remote binary
+for each platform (win/linux/mac). Plan is to hook it with quay github build trigger to generate this image
+for each PR merged (for any branch or tag).
+
+User can copy those binary on specific platform using following
+
+- For Windows binary
+```bash
+$ podman cp $(podman create --name remote-temp quay.io/containers/podman-remote-artifacts:latest):/windows/podman.exe . && podman rm remote-temp
+```
+
+- For Linux binary
+```bash
+$ podman cp $(podman create --name remote-temp quay.io/containers/podman-remote-artifacts:latest):/podman-remote-static . && podman rm remote-temp
+```
+
+- For Mac binary
+```bash
+$ podman cp $(podman create --name remote-temp quay.io/containers/podman-remote-artifacts:latest):/darwin/podman . && podman rm remote-temp
+```
```

Comment:
Oh! nevermind my last comment, the above can easily be accommodated in our build script.  It has [an image-mod hook available](https://github.com/containers/automation/tree/main/build-push#use-in-automation-with-modified-images) (docs) that can produce these extra image flavors prior to pushing.

For reference, [we currently use a version-tagging mod-script](https://github.com/containers/automation_images/blob/main/build-push/bin/tag_version.sh) to handle creation of extra tags for the `stable` podman images.

Reply:
(I can do this work later, no need to make this PR depend on it.